roles/network/tasks/rpi_debian.yml

```diff
@@ -20,11 +20,16 @@
     dest: /etc/dhcpcd.conf
     src: network/dhcpcd.conf.j2
 
-- name: Copy the bridge script for RPi
+- name: Copy the neworking script for RPi
```

Comment:
@georgejhunt typo..plz change "neworking" to "networking" on Line 23 of roles/network/tasks/rpi_debian.yml